### DISCUSSION TITLE
UpdateDTO中的@NotNull不生效，@NotEmpty是生效的

### DISCUSSION THREAD

**Discussion by 18285160741** [zuihou/zuihou-admin-cloud#82] [02.提问]
UpdateDTO中的id不能为空，@NotNull 校验未生效。
直接提交成功了。
@NotEmpty是生效的

![image](https://user-images.githubusercontent.com/37412616/109788010-cb49cb80-7c49-11eb-85d7-89b225746ce2.png)

请问大佬这应该怎么处理呢？谢谢

**Comment by zuihou** [comment]
![image](https://user-images.githubusercontent.com/8500410/109895577-684c4900-7cca-11eb-958b-6a42ff4a734c.png)

![image](https://user-images.githubusercontent.com/8500410/109895644-89149e80-7cca-11eb-9871-f7a23241a3ee.png)

**Reply by zuihou** [reply]
无法重现
